common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -123,6 +124,10 @@ public static String getDelimitedDefaultScopeString() {
     public abstract AcquireTokenResult acquireToken(final InteractiveTokenCommandParameters request)
             throws Exception;
 
+    public AcquireTokenResult acquireTokenForAccountTransferV2(final AccountTransferV2TokenCommandParameters request) throws Exception {
```

Comment:
Why not declare it abstract here like we are doing with the rest of the methods?

Reply:
If i declare it abstract here, i'd need to define it in every class that extends BaseController, and have them all (except BrokerMsalController) throw an unsupported method exception. This seemed cleaner to do it this way, and just override it in BrokerMsalController.